R/data_interface.R

```diff
@@ -58,6 +63,8 @@ spark_read_csv <- function(sc,
                            name,
                            path,
                            header = TRUE,
+                           columns = NULL,
+                           infer_schema = FALSE,
```

Comment:
`infer_schema` needs to be set to `TRUE` to maintain compatibility with the existing API.